server/datastore/mysql/migrations/tables/20220524102918_CleanupOrphanedPolicyMemberships.go

```diff
@@ -0,0 +1,32 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220524102918, Down_20220524102918)
+}
+
+func Up_20220524102918(tx *sql.Tx) error {
+	delStmt := `
+    DELETE pm
+    FROM policy_membership pm
+    LEFT JOIN policies p ON p.id = pm.policy_id
```

Comment:
I believe there's no need for `LEFT` here as `policy_membership` has a foreign key to `policies` (`CONSTRAINT policy_membership_ibfk_1 FOREIGN KEY (policy_id) REFERENCES policies (id) ON DELETE CASCADE`)